app/src/main/java/com/example/parkingsystemjava/mvp/contract/ParkingReservationDialogContract.java

```diff
@@ -0,0 +1,10 @@
+package com.example.parkingsystemjava.mvp.contract;
+
+public interface ParkingReservationDialogContract {
```

Comment:
if you are not using this now, it should not be here. Please remove it

Reply:
@mauroLonghi told me to leave like that with a comment. should i remove it anyways?